pydra/tasks/fsl/preprocess/susan.py

```diff
@@ -2,6 +2,31 @@
 from pydra import ShellCommandTask
 import typing as ty
 
+
+def format_usans(field: ty.Sequence[ty.Tuple[str, float]] = None) -> str:
+    """Format usans argument to its appropriate argstr.
+
+    Examples
+    --------
+    >>> format_usans([])
+    '0'
+    >>> format_usans([('/path/to/file', 2.5)])
+    '1 /path/to/file 2.5'
+    >>> format_usans([('/path/to/file1', 10.5), ('/path/to/file2', 22.1)])
+    '2 /path/to/file1 10.5 /path/to/file2 22.1'
+    """
+    try:
+        n_usans = len(field)
+    except TypeError:
+        n_usans = 0
+
+    if n_usans > 0:
+        usan_bt = " ".join([f"{usan} {bt}" for usan, bt in field])
+        return f"{n_usans} {usan_bt}"
+    else:
+        return f"{n_usans}"
```

Comment:
We should get a `TypeError` if we don't get a sequence. The function is annotated; might as well get a useful traceback instead of a mysterious "0".

```suggestion
    return " ".join([f"{len(field)}"] + [f'{usan} {bt}' for usan, bt in field])
```